src/app/product/product.cpp

```diff
@@ -0,0 +1,50 @@
+#include "product.hpp"
+
+Product::Product(const std::string &name, const double price, const uint amount)
+    : name_(name), price_(price), amount_(amount)
+{
+}
```

Comment:
Please use initializer list in constructor

Reply:
Done